### ISSUE TITLE
Draconic Bow Dupe

### ISSUE THREAD

**Issue by LezChap** [brandon3055/Draconic-Evolution#334]
## Description
As far as I've been able to test, it ONLY works with (and dupes) Draconic Bows.  I've tested with everything else I could think of (including Wyvern Bow)...so this isn't a major dupe.

It requires an autonomous activator, EnderIO Conduits (only conduit I can think of that can filter based on charge level) with some charging filters, and charging device of some sort (I used EnderIO Creative Capacitor, but it works with the charging device in your mod too), a Draconic Bow, and an Infinity Enchanted Book.

Give the Draconic Bow Infinity, and set the Autofire value to "On".   Set up the Autonomous Activator to the grey setting on the side you want to import/export from via the conduits.  Set up the conduit to pull out when charge is <100%, and to accept when charge is 100%.  The other end of the conduit should be input/export into whatever your charging device of choice is (I used a creative capacitor from EnderIO).  Place the Draconic Bow into the autonomous activator, and every time it fires, it'll dupe the Bow.  A similar setup to what I've done can be seen in Direwolf20's video: https://youtu.be/PLuOCqOFqAo?t=1598  Soaryn goes on to explain what he believes is happening to cause the item to dupe.

## Base information
* Minecraft version: 1.7.10
* Minecraft Forge version: 10.13.4.1614
* Draconic Evolution version: 1.0.2d (I know there's a couple newer versions, but the changelogs don't list any changes to the Bow so I didn't test them)

## Mod List or Mod Pack (include version)
I was testing in SkyFactory 2.5.2 (FTB/Curse launchers, default mods)

**Comment by brandon3055** [comment]
Items themselves have absolutely no control over how they are handles by inventories and as far as firing my bows work the same as any other bow. So this is most likely an issue with the activator or the item conduit. 

Note: This dose also work with wyvern bows although i only got it to work once.

Edit: I think this confirms it! I was able to recreate this dupe with the bow of bone from thaumcraft.
[http://puu.sh/pozhD/42352ef2e9.png](url)
I did this using an advanced item filter set to pull out the bow when it has a damage value of exactly 1. Then i place a bow with a damage value of 1 in the activator and it gets pulled out twice.

Edit2: Dosnt even need to be damaged to work.

Edit3 You should have done more testing... 
[http://puu.sh/pozwK/08f6b23ccd.png](url)
[http://puu.sh/pozGu/61ef8883fa.png](url)
[http://puu.sh/pozPq/fcbb21f543.pnge](url)
[http://puu.sh/pozT4/0b76cab689.png](url)

So basically whenever an item gets pulled out by an advanced item filter (or energy item filter) while its being used by the activator it gets duped.
